src/Components/Icons/Icons.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCode } from "@fortawesome/free-solid-svg-icons";
```

Comment:
you can import all of the together like:
import 
{faCode, faFolderOpen} from ''"

Reply:
Done.